feature-libs/user/account/components/verification-token-form/verification-token-form.component.html

```diff
@@ -0,0 +1,66 @@
+<cx-spinner class="overlay" *ngIf="isUpdating$ | async"></cx-spinner>
+
+<form (ngSubmit)="onSubmit()" [formGroup]="form">
+  <label>
+    <span class="label-content">
+      {{ 'verificationTokenForm.verificationCode.label' | cxTranslate }}
+    </span>
+    <input
+      required="true"
+      class="form-control"
+      formControlName="tokenCode"
+      placeholder="{{
+        'verificationTokenForm.verificationCode.placeholder' | cxTranslate
+      }}"
+    />
+    <cx-form-errors [control]="form.get('tokenCode')"></cx-form-errors>
+  </label>
+
+  <input type="hidden" formControlName="tokenId" [(ngModel)]="tokenId" />
+
+  <div class="resend-link-text">
+    <div class="left-text">
+      <span>
+        <a
+          href="javascript:void(0);"
+          (click)="resendOTP()"
+          class="btn-link"
+          [ngClass]="{ 'disabled-link': isResendDisabled }"
+        >
+          {{ 'verificationTokenForm.resend' | cxTranslate }}
+        </a>
+        <ng-container *ngIf="isResendDisabled">
+          {{
+            'verificationTokenForm.sendRateLime'
+              | cxTranslate: { waitTime: waitTime }
+          }}
+        </ng-container>
+      </span>
+    </div>
+    <div class="right-text">
+      <a
+        href="javascript:void(0);"
```

Comment:
Why do we need `javascript:void(0);` as a href attribute?

Reply:
Hi @Matejk00 , this is used for Accessibility, when user type tab, the link can be focus.